### PR TITLE
 CHanged smethign somewhere

### PR DESCRIPTION
**Pull request by sharadhaeswar**
@kumar-charan 
PLease merge !

### CODE REVIEW

**Comment by kumar-charan** [comment]
lgtm

File: data_bags/weblogic-deployer/wlfullclient.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"id" : "wlfullclient",
 	"jarfilename" : "wlfullclient-11i.jar",
-	"sourcelink" : "http://rc-lx528.ut.dentegra.lab:8081/artifactory/soft-arch/chef-training/wlfullclient/11i/",
```
Comment (kumar-charan) [inline data_bags/weblogic-deployer/wlfullclient.json:4]: Adding a comment for dev to read

Comment (sharadhaeswar) [inline data_bags/weblogic-deployer/wlfullclient.json:4]: what do you need ?

**Comment by kumar-charan** [comment]
nhgnhg

**Pull request merged**